_dashboards/dashboard/plugins-dashboards.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Integrating plugins into a dashboard
+parent: Observability
+nav_order: 5
+---
+
+# Integrating plugins into a dashboard
+
+Introduced 2.7
```

Comment:
This label is not needed. Typically, we use it for APIs but not for other types of documentation.

Reply:
Deleted label